docs/reference/commandline/run.md

```diff
@@ -981,9 +981,57 @@ $ docker run --restart=always redis
 This will run the `redis` container with a restart policy of **always**
 so that if the container exits, Docker restarts it.
 
-You can find more detailed information on restart policies in the
-[Restart Policies (--restart)](../run.md#restart-policies---restart)
-section of the Docker run reference page.
+When a restart policy is active on a container, it will be shown as either `Up`
```

Comment:
We should avoid future tense here.


Also; this commit mentions "docs: move --restart to docker run reference", but I don't see the content being removed from the other document in this commit 🤔

Reply:
oops